bin/packer-build-trigger

```diff
@@ -24,7 +24,7 @@ fi
 
 set -o xtrace
 exec travis-packer-build \
-  --chef-cookbook-path="${CHEF_COOKBOOK_PATH}" \
+  --chef-cookbook-path="${CHEF_COOKBOOK_PATH} ${TOP}/.git::cookbooks" \
```

Comment:
is `${TOP}` from [this](https://github.com/travis-ci/packer-templates/blob/bfb94921f954cc083ed3cec3179c45b254f8d6c6/bin/packer-build-trigger#L5)?